src/libNode/PoWProcessing.cpp

```diff
@@ -281,6 +281,40 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
   return true;
 }
 
+bool Node::CheckIfGovProposalActive() {
```

Comment:
> Looks like only current setup will work with only one proposal being active in our network. @chetan-zilliqa @ansnunez
> `m_govProposalInfo` is tied to node which track only one proposal until its lifetime.
> 
> Is this expected behaviour?

On node side yes we support one proposal vote only inside PoW submission. If a miner wants to vote on another proposal it would have to be on a different PoW submission by calling `SetVoteInPow` again.